### PR TITLE
wavpack: 5.4.0 -> 5.5.0

### PR DESCRIPTION
**Pull request by yorickvP**
CVE-2021-44269

#166706

###### Description of changes

[Changelog](https://www.wavpack.com/changelog.txt)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/wavpack/default.nix [hunk 1]
```diff
@@ -2,18 +2,20 @@
...
   pname = "wavpack";
-  version = "5.4.0";
+  version = "5.5.0";
 
   enableParallelBuilding = true;
 
   nativeBuildInputs = [ autoreconfHook ];
   buildInputs = lib.optional stdenv.isDarwin libiconv;
 
+  preAutoreconf = "touch config.rpath";
```
Comment (mweinelt) [inline pkgs/development/libraries/wavpack/default.nix:12]: Leave a note why that is required.

**Comment by risicle** [comment]
Builds happily for me, macos 10.15 & nixos x86_64, builds `mlt` on nixos x86_64. `pkgsMusl`, `pkgsStatic`, `pkgsCross.aarch64-multiplatform variants build for me.

**Comment by yorickvP** [comment]
Updated to get the config.rpath from gettext, like [wavpack's autogen.sh](https://github.com/dbry/WavPack/blob/master/autogen.sh) does

**Review by risicle — APPROVED** [review]
Re-tested same targets successfully :point_up:

**Pull request merged**